patches/chrome-browser-ui-browser.cc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
+index 6e0ce3191b697c1f45ae9fd5b73679f50c131cd5..b46546f5a8c511ff524902e3d13c69528de27ee4 100644
+--- a/chrome/browser/ui/browser.cc
++++ b/chrome/browser/ui/browser.cc
+@@ -555,6 +555,9 @@ Browser::~Browser() {
+   // because the InstantController destructor depends on this profile.
+   instant_controller_.reset();
+ 
++  // Clear browsing data on exit, if needed.
++  content::BraveClearBrowsingData::ClearOnExit(profile_);
```

Comment:
we don't need to patch in a new method here, we can do a chromium_src override for profiles::RemoveBrowsingDataForProfile

Reply:
sorry, I mean DestroyProfileWhenAppropriate